web/html/src/manager/proxy/container-config-messages.tsx

```diff
@@ -0,0 +1,52 @@
+import * as React from "react";
+
+import { Messages } from "components/messages";
+
+type SuccessType = boolean | undefined;
+type MessagesType = React.ReactNode[];
+
+export const ContainerConfigMessages = (success: SuccessType, messagesIn: MessagesType, loading: boolean) => {
+  var messages: React.ReactNode = null;
```

Comment:
Personally I would omit this variable if it's only used to return it right away and nothing else, but that's purely a taste preference.  

```tsx
if (success) {
  return <Messages .../>
} else if (...) {
  return ...;
}
return null;
```